pages/common/zip2john.md

```diff
@@ -0,0 +1,17 @@
+# zip2john
+
+> A tool to extract password hashes from zip files for use with John the Ripper password cracker.
+> This is a utility tool usually installed as part of the John the Ripper installation.
+> More information: <https://www.openwall.com/john/>.
+
+- Extract the password hash from an archive, listing all files in the archive:
+
+`zip2john {{path/to/file.zip}}`
+
+- Extract the password hash using [o]nly the specified file within the archive:
+
+`zip2john -o {{compressed_file_in_zip}} {{path/to/file.zip}}`
+
+- Output the extracted hash into a file so that it can be used with John the Ripper:
```

Comment:
Instead of this example why not a direct pipe to the John the Ripper app?

Reply:
Since there are many different formats, typical process is to read the first few characters of the hash to determine encryption format which will determine what option to pass to the John the Ripper app in the `--format=` flag.  I figured that detail is more within John the Ripper domain of instruction.

Reply:
```suggestion
- Extract the password hash from a compressed file to a specific file (for use with John the Ripper):
```